tests/test_sequence_classification.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+. ./tests/lib.sh
+
+genienlp train --train_tasks ood_task --model TransformerForSequenceClassification --pretrained_model distilbert-base-uncased --save $workdir/model --train_iterations 4 --save_every 2 --log_every 2 --val_every 2 --data $SRCDIR/dataset/ood_task/ --skip_cache --force_fast_tokenizer --train_batch_tokens 200
```

Comment:
Can you please add a test for prediction too and checking against expected results. Again you can look at other tests for how it's done. It will ensure the later changes we make to the code do not change the output unexpectedly.